xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamFile.cpp

```diff
@@ -142,12 +142,15 @@ BitstreamStats CDVDInputStreamFile::GetBitstreamStats() const
     return m_stats;
 }
 
+// Use value returned by filesystem if is > 1
+// otherwise defaults to 64K
 int CDVDInputStreamFile::GetBlockSize()
 {
-  if(m_pFile)
-    return m_pFile->GetChunkSize();
-  else
-    return 0;
+  int chunk = 0;
+  if (m_pFile)
+    chunk = m_pFile->GetChunkSize();
+
+  return ((chunk > 1) ? chunk : 64 * 1024);
```

Comment:
Shouldn't this be `> 0`?

Reply:
0 and 1 have special meanings:

0 --> Chunks cannot be used but in our case it is rather "not set" (because it is the default value) and most code don't overrides default.

1 --> File can be read with chunks and these can have any size.

another value "xx" --> Must be read with chunks of at least xx bytes. This is usually a legacy of older hardware which usually requires reading in amounts that matched the physical size of the sector (e.g. DVD's or CD-ROM block_size = 2048 bytes). You can not read less that one sector.

Then "1" means ---> is possible read in chunks of any size ---> 64K is fine